components/mediation/mediators/dataservices-mediator/org.wso2.micro.integrator.mediator.dataservice/src/main/java/org/wso2/micro/integrator/mediator/dataservice/DataServiceCallMediator.java

```diff
@@ -515,4 +508,12 @@ public void setParamExpression(SynapsePath paramExpression) {
             this.paramExpression = paramExpression;
         }
     }
+
+    public class OperationsType {
+
+        public static final String SINGLE = "single";
```

Comment:
Let's move these to DataServiceCallMediatorConstants class

Reply:
Fixed in 915b0bc